flexmeasures/ui/templates/views/sensors.html

```diff
@@ -33,6 +33,51 @@
               <div class="alert alert-info" id="dstwarn" style="display:none;"></div>
               <div class="alert alert-info" id="sourcewarn" style="display:none;"></div>
           </div>
+          <style>
```

Comment:
Let's find a place in flexmeasures.css for these styles.

Reply:
Looks like "Select Dates" button is linked to the chart and since we want the table with info to be on the top, "Select Dates" button is moves down, same as a chart